src/main/java/org/gridsuite/study/server/networkmodificationtree/AbstractNodeRepositoryProxy.java

```diff
@@ -232,12 +243,16 @@ public UUID getVoltageInitResultUuid(UUID nodeUuid) {
         return getVoltageInitResultUuid(getNode(nodeUuid));
     }
 
-    public void updateBuildStatus(UUID nodeUuid, BuildStatus buildStatus, List<UUID> changedNodes) {
-        updateBuildStatus(getNode(nodeUuid), buildStatus, changedNodes);
+    public void updateBuildStatus(UUID nodeUuid, BuildStatus buildStatusGlobal, BuildStatus buildStatusLocal, List<UUID> changedNodes) {
+        updateBuildStatus(getNode(nodeUuid), buildStatusGlobal, buildStatusLocal, changedNodes);
+    }
+
+    public BuildStatus getBuildStatusGlobal(UUID nodeUuid) {
```

Comment:
getBuildStatusGlobal -> getGlobalBuildstatus
getBuildStatusLocal -> getLocalBuildStatus ?

Reply:
Fine by me, I've done the change across the PR